src/app/project/add.tpl.html

```diff
@@ -39,6 +39,7 @@
                        ng-model="vm.project_name"
                        ng-model-options="{ debounce: 500 }"
                        project-name-available-validator
+                       organization-id="vm.currentOrganization.id"
```

Comment:
Can you remember the case in 8fb5e56, on how to replicate in order to check if the `timeout` part will be needed in this particular case?